docs/reference/taxii-feeds.md

```diff
@@ -1,7 +1,32 @@
 # Taxii feeds
```

Comment:
Can you change the title to TAXII feeds

Reply:
I'd say you could change the title to something like "TAXII management" to encompass all that revolves around TAXII. This will allow to add the "data sharing" part to your doc. You don't need to write a lot, you can redirect to https://docs.opencti.io/latest/usage/feeds/#taxii-collections

The important thing to explain on this page is not the configuration of an import or export flow (which is explained on other pages to which we redirect), but how it works:
- A taxii collection shares the data that matches the filters AND to which the user has access. It only shares the entities that match the filters, not its relations (unlike the Live stream).
- Is it real time? Or does it retrieve every x amount of time?
- Explain the "current status" -> This is the date of the last entity imported and therefore the state of progress of the import. It could be interesting to see whether it's on today's date or whether it's stuck on a previous date (possible bug in this case).
- Other interesting mechanisms to explain ?
You can also refer to complementary resources, such as the TAXII doc from oasis.

We have an internal knowledge page on our Notion that could help you